app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/PagerPageHolder.kt

```diff
@@ -241,8 +252,27 @@ class PagerPageHolder(
             .subscribeOn(Schedulers.io())
             .observeOn(AndroidSchedulers.mainThread())
             .doOnNext { isAnimated ->
-                if (viewer.config.dualPageSplit) {
-                    openStream = ImageUtil.dualPageSplit(openStream!!)
+                if (!isAnimated && viewer.config.dualPageSplit) {
```

Comment:
Some manga use static gif as image format, so `isAnimated` should be removed.
<img width="307" alt="截屏2021-01-08 18 40 34" src="https://user-images.githubusercontent.com/32921531/104006130-0ba84280-51e1-11eb-9743-012c46a32dfa.png">

Reply:
Hmm, I added `!isAnimated` due to concern that the reader would crash if it tried to split an animated GIF.

Reply:
Thanks for your pull request and replying. I am trying figuring out what happends behind the code, which may cost a few hours.

Reply:
No problem. If you find eventual problems with the code, you could comment where you think the issue is. 
Also, you can always modify the code afterward if you don't like a certain part of the code or think that it can be done in a different way